userspace/falco/app/actions/load_config.cpp

```diff
@@ -61,9 +61,12 @@ falco::app::run_result falco::app::actions::load_config(const falco::app::state&
 	if (!s.options.conf_filename.empty())
 	{
 		falco_logger::log(falco_logger::level::INFO, "Falco initialized with configuration files:\n");
-		for (const auto& path : loaded_conf_files)
+		for (const auto& pair : res)
 		{
-			falco_logger::log(falco_logger::level::INFO, std::string("   ") + path + "\n");
+			auto config_path = pair.first;
+			auto validation = pair.second;
+			auto priority = validation == yaml_helper::validation_ok ? falco_logger::level::INFO : falco_logger::level::WARNING;
+			falco_logger::log(priority, std::string("   ") + config_path + " | validation: " + validation + "\n");
```

Comment:
Example output:
```
Wed Aug 21 15:43:33 2024: Falco initialized with configuration files:
Wed Aug 21 15:43:33 2024:    ../falco.yaml | validation: ok
```

Reply:
Note that just a warning is emitted, no fatal error is thrown.

Reply:
Why only a warning and no error on validation issue ?
If it's optional, should we also have a strict mode that will trigger an exception ?

Reply:
Fair enough